lib/pure/pegs.nim

```diff
@@ -20,11 +20,11 @@ include "system/inclrtl"
 const
   useUnicode = true ## change this to deactivate proper UTF-8 support
 
-import
-  strutils
+import strutils, macros
 
 when useUnicode:
   import unicode
+  # export unicode.`==`
```

Comment:
Small nitpick: If you want to remove this code then just remove it, don't comment it out.

Reply:
I don't really want to remove the line yet: commit 6c092fe broke two of the tests in the *method* category in the Travis and Appveyor CI tests (think it was *tmultim2* and *tmultim6*). Commits 5b08c62 and 5301817 were just a test to check whether commenting out the *export unicode.`==`* would unbreak this. They did, but I don't understand why: is there something strange going on with exports and dynamic linking (see the *rtl* and *extern* pragmas of *rawMatch*)? Also, I can't find a *bind* statement to make this last wretched *export* go away :o/

So the commented-out line was meant as a reminder that this is code change is not really a solution, just a test. Should have made this clear in the comment.

Reply:
@gemath Are you sure that it was your code that broke the method tests?
  
Because on devel it was this https://github.com/nim-lang/Nim/commit/75ef90da828151fa2426b3cd757da03062e026d4 commit which broke them and it was fixed later by https://github.com/nim-lang/Nim/pull/8249.

Reply:
Ah, then it was just coincidence, thx. But I still want to get rid of that damned export..